app/src/main/java/de/westnordost/streetcomplete/screens/user/profile/ProfileFragment.kt

```diff
@@ -164,15 +176,19 @@ class ProfileFragment : Fragment(R.layout.fragment_profile) {
             val shouldShow = statistics.rank != null && statistics.rank > 0 && statistics.count > 50
             val countryLocale = Locale("", statistics.countryCode)
             binding.localRankContainer.isGone = !shouldShow
-            binding.localRankText.text = "#${statistics.rank}"
-            binding.localRankLabel.text = getString(R.string.user_profile_local_rank, countryLocale.displayCountry)
+            if (shouldShow) {
+                binding.localRankText.text = "#${statistics.rank}"
+                binding.localRankLabel.text = getString(R.string.user_profile_local_rank, countryLocale.displayCountry)
+                binding.localRankText.background = LaurelWreathDrawable(resources, min(100 - statistics.rank!!, 100))
```

Comment:
uhh, can statistics.rank be null?

Reply:
in such case https://github.com/streetcomplete/StreetComplete/pull/4203/files/91a1e78eda50992bad7a3ce4f4b2bbc236aa82a5#diff-a4897d401da7827ccc0867cd49c938294ee4c49d795d6a3173017c9ecbcf03b9L164 will set `shouldShow` to false